config/test.yaml

```
@@ -8,7 +8,8 @@ redis:
   url: redis://localhost:6379
 
 metrics:
-  sink_address: "0.0.0.0:0"
+  sink_host: "0.0.0.0"
+  sink_port: 0
```

I think that this should default to `8125`. `0.0.0.0` is a sort-of-sensical port for this to be on, but port 0 definitely doesn't make sense here.
```suggestion
  sink_port: 8125
```